src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/ActivityFunction/FunctionAnalyzer.cs

```diff
@@ -147,9 +147,9 @@ private string GetQualifiedTypeName(ITypeSymbol typeInfo)
                     return typeInfo.ToString();
                 }
 
-                if (!typeInfo.Name.Equals(""))
+                if (string.IsNullOrEmpty(typeInfo.Name))
```

Comment:
I think the condition might be backwards. Should this be `if (!string.IsNullOrEmpty(typeInfo.Name))`?